src/app/book/pages/book-list/book-list.component.html

```diff
@@ -0,0 +1,24 @@
+<div class="buttons-container-admin" *ngIf="admin">
+  <button class="button">
```

Comment:
why do u need button here?
can't it be just a link?

Reply:
fixed